posts/bayesian-model-comparison/index.qmd

```diff
@@ -417,62 +426,78 @@ However if we can specify costs then decision making based on these would be the
+  f"Ratio {model_2} / {model_1} is {round(model_2_time / model_1_time, 2)}"
+)
+```
+
+
+That is, MLP is a lot slower to fit than RF.
 There are three decisions we can make 
 
-1) SVM is better than RF
-2) SVM is equivalent to RF
-3) SVM is worse than RF
+1) RF is better than MLP
+2) RF is equivalent to MLP
+3) RF is worse than MLP
 
-Consider the following cost-matrix:
+We consider the following cost-matrix:
 
 +---------------+---------------+--------------------+--------------------+
-|               | SVM is better | are equivalent     | RF is better       |
+|               | RF is better  | are equivalent     | MLP is better      |
 +===============+===============+====================+====================+
-| Choose SVM    | 0             | -5                 | 2                  |
+| Choose RF     | 0             | -5                 | 2                  |
 |               |               |                    |                    |
 +---------------+---------------+--------------------+--------------------+
-| Choose RF     | 7             | 5                  | 0                  |
+| Choose MLP    | 7             | 5                  | 0                  |
 |               |               |                    |                    |
 +---------------+---------------+--------------------+--------------------+
 
 where the $(i, j)$th entry is the cost incurred by making decision $i$ when $j$ is correct.
-Here we have a $2 \times 3$ matrix as we only consider the options of selecting the SVM or the RF, no abstention, or anything else.
+Here we have a $2 \times 3$ matrix as we only consider the options of selecting either one model or the other, no abstention, or anything else.
 
 In this case we have:
 
-* cost of choosing SVM is:
-  + 0 if SVM is better, 
+* cost of choosing the Random Forest is:
+  + 0 if it is better, 
   + -5 if they are equivalent as we save compute time, 
-  + 2 if RF is better as we would lose performance
+  + 2 if MLP is better, as we would lose performance
 
-* cost of choosing RF is:
-  + 7 if SVM is better because we pay for computational and performance cost, 
+* cost of choosing the MLPClassifier is:
+  + 7 if RF is better because we pay for computational and performance cost, 
   + 5 if they are equivalent as we add to the compute time, 
-  + 0 if RF is indeed better
-
+  + 0 if MLP is indeed better
 
 The *expected cost* can then be obtained by multiplying the cost_matrix with the relevant posterior probabilities. 
-In this case, the relevant probabilities are $P(M_1 > M_2 | D)$, $P(M_1 \sim M_2 | D)$ and $P(M_1 < M_2 | D)$.
+In this case, the relevant probabilities are $P(RF > MLP | D)$, $P(RF \sim MLP | D)$ and $P(RF < MLP | D)$.
 
 ```{python}
 cost_matrix = np.array([
   [0., -5., 2.],
   [7., 5., 0.],
 ])
 
-probabilities = np.array([prob_svm_better, prob_rope, prob_rf_better])
+probabilities = np.array([prob_model_1_better, prob_rope, prob_model_2_better])
```

Comment:
Maybe better to wrap this as on rendering requires scrolling sideways to read it all.

Reply:
Actually if we're sticking with RF and MLP maybe we just call them that instead of model 1 and 2 and maybe that makes it short enough.

Reply:
it was only 80 chars actually, and i think overflow very dependent on peoples screen set up, but i've split over lines. easier than renaming, and achieves same goal i think.